helpers/renderCopyButtons.ts

```diff
@@ -92,7 +91,10 @@ const renderCopyButtons = (
     document.current?.querySelectorAll("div.remark-highlight") ?? [];
 
   codeSnippets.forEach((codeSnippet) => {
-    if (codeSnippet.childNodes.length === 2) {
+    if (
+      codeSnippet.childNodes.length === 2 ||
+      !(codeSnippet instanceof HTMLElement)
+    ) {
```

Comment:
@ntorbinskiy What's this piece of code is doing?

Reply:
2 is a magic number

Reply:
Because in React 18, useEffect, even without any dependency's, work 2 times. So, I put in it check, because If I don't- I have 2 copy buttons

Reply:
Anyway, remove all magic numbers